### ISSUE TITLE
[Minor Enhancement] Highlight chosen option in all dropdown menu briefly

### ISSUE THREAD

**Issue by dmint789** [TeamNewPipe/NewPipe#8873]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Please make it so that when an item in a dropdown list (like the one that comes up when you long-press on a video) is tapped, it is visibly highlighted for a brief moment to provide some feedback to the user that they didn't accidentally tap on the wrong option, like in other apps.

### Why do you want this feature?

This would slightly improve the user experience

**Comment by u7145232** [comment]
Hi, I would like to work on this issue :)

**Comment by opusforlife2** [comment]
Did you intend to work on one issue or multiple? I'm assigning you here, but if you don't want it, please leave a comment mentioning that.

**Comment by u7145232** [comment]
I've run the app through a couple emulators and it seems like there is already a brief highlight on the dropdown menu items mentioned in this issue?
![image](https://user-images.githubusercontent.com/110881529/197508961-380ceb0d-bb0e-45c3-9c2a-ef0dd0dc9a88.png)

**Comment by dmint789** [comment]
It might work differently in the emulator. On my phone whenever I select an option it just instanly closes the dropdown and I am not 100% sure what I actually selected.

**Comment by u7145232** [comment]
Thanks for reply! It's hard for me to see this issue since I'm on the emulator but is this also an issue on the downloads resolution menu?
![image](https://user-images.githubusercontent.com/110881529/197519707-90209a22-9680-4b8a-88b0-776e09c32db0.png)
Thanks!

**Comment by dmint789** [comment]
It is, although here you can immediately see what you selected anyways. I think I just realized what the issue is. It actually does play an animation when you select an item from dropdowns, but this animation is very slow. I can only see it play if I hold down the option I selected. It should probably just be an immediate highlight instead of this slow animation, because I don't think users tend to hold down dropdown options before they select them.

**Comment by u7145232** [comment]
Does the animation play sufficiently fast in any other menus?

**Comment by dmint789** [comment]
I don't think so. I imagine it's the same everywhere. If you have any specific dropdown menus in mind, I can test those for you

**Comment by u7145232** [comment]
Can you confirm if the lag is in all touch items/all items that have the animation effect? I don't really have any specific ones in mind at the moment. Thanks!

**Comment by opusforlife2** [comment]
@u7145232 You could always look for another issue if this one isn't panning out.

**Comment by u7145232** [comment]
I've added a custom highlight effect to replace the current one and I think it does appear more consistently and is also more noticeable, might help with accessibility.

**Comment by opusforlife2** [comment]
Screenshot?

**Comment by u7145232** [comment]
Will do! I just need to apply it to the long press menu. Seems that the menu item highlights are different for the long press menu than from other ones so I'm having a some trouble finding it. Would you be able to point me to it? @opusforlife2 
Thank you!

**Comment by u7145232** [comment]
This issue seems to be a lot more widespread than I first thought, as the author mentioned it seems to apply to all selectable buttons not just dropdown menus which I'm having difficulty locating and seems more like a bug fix than a feature request. The issue is also difficult for me to replicate on the emulator so I don't think I'll be able to do this unfortunately :(

**Comment by opusforlife2** [comment]
@u7145232 ANU student?

**Comment by dmint789** [comment]
Any updates on this issue?
